run/o2aod_mc_to_hepmc.cxx

```diff
@@ -315,16 +132,20 @@ struct Task3 {
    * command line argument (@c --hepmc-aux) rather than to rely on an
    * auto-generated name (would be @ --processAux).
    */
-  decltype(o2::framework::ProcessConfigurable{&Task3::processAux,
-                                              "hepmc-aux", false,
-                                              "Process auxilirary "
+  decltype(o2::framework::ProcessConfigurable{&Task::processPlain,
```

Comment:
Since we have c++20 now, one can use `auto` here instead of an explicit type. The old trick is due to the fact that type deduction did not work for struct members.

Reply:
Yeah. I know you can do `auto` here with C++20, but you couldn't until very recently.  BTW, has #12732 been fixed?  Until that is fixed, it does not make much sense to prevent compilation with C++17.

Reply:
The defaults in build system have been moved to c++20 for a while now, setting in CMakeLists.txt is irrelevant. It should be updated, of course, you can create a PR.

Reply:
Really?  Do you need a PR for that?  Just do the edit and get it done with, seeing someone clearly forgot.

Reply:
Hmm.  Using `auto` as type of `doPlain` didn't work 

```
/opt/sw/src/AliceO2/run/o2aod_mc_to_hepmc.cxx:135:3: error: non-static data member declared with placeholder ‘auto’
  135 |   auto doPlain = o2::framework::ProcessConfigurable{&AodToHepmc::processPlain,
      |   ^~~~
```

using the full `decltype` code.